README.md

```diff
@@ -45,40 +59,115 @@ If you're developing on the code, you'll use `make`.
 
 #### Required tools:
 
-- `java` (tested on JDK8)
-- `maven`
+- `java` (Oracle or OpenJDK 8)
+- `maven` or `gradle`
 
 
-#### Optional tools:
+## Services
 
-Steps:
+After you've setup the SDK and client object, you can access a given service via a top-level function named after the
+service. For instance, the Shop API is available at `client.shop()`.
 
-- `git clone [...] && cd [project root]`
-- `git submodule update --init --remote`
-- `make`
+The object handed back by this method call is structured with callable methods for each API call available for the given
+service. Below you'll find some samples.
 
 
-#### Debug mode
+### Shop API
 
-Add a note here about debug mode, and how to activate it.
+The Shop API is what powers online ordering services with Bloombox. It enables features for **managing and operating an
+integrated digital storefront**, with user signup, login, support for hours, zipcode verification, and full on pickup or
+delivery ordering orchestration.
 
+#### Getting hours info
 
-## Services
+Each Bloombox digital storefront maintains a set of hours that the user can control. Using the *info()* method, an
+integrating system can check to see the current status of the storefront, according to those hours:
 
+###### Java
+```java
+  // with our client object, obtain shop info synchronously
+  final ShopInfo.Response infoResponse = client.shop().info();
+  if (infoResponse.getStatus() == ShopStatus.OPEN) System.out.println("The shop is OPEN.");
+```
 
-### Shop API
+###### Kotlin
+```kotlin
+  // with our client object, obtain shop info synchronously
+  val info = client.shop().info()
+  if (info.getStatus() == ShopStatus.OPEN) print("The shop is OPEN.")
+```
+
+#### Enumerated shop statuses
+
+According to the current set of _regular hours_ (recurring hours rules that apply everyday, on weekdays, weekends, or
+specific days of the week), and _special hours_ (hours for specific dates, like _New Year's Day_ or _Thanksgiving_), a
+digital storefront may take on the following statuses:
 
-Demo code coming soon.
+##### Shop statuses
+- `OPEN`: The storefront is open for any and all configured order types.
+- `DELIVERY_ONLY`: The storefront is currently open only for delivery orders.
+- `PICKUP_ONLY`: The storefront is currently open only for pickup orders.
+- `CLOSED`: The storefront is currently closed and not accepting orders of any type.
+
+When an order is submitted to a shop that is `CLOSED` (or a `PICKUP` order is submitted during `DELIVERY_ONLY`, or a
```

Comment:
Are we planning on implementing functionality so that an order can still be placed/queued during one of these times with a "future fulfillment" that goes through when that order type is accepted again?

Reply:
that's a great point. at the moment orders are just closed depending on the _current_ store status. we should file as a bug and fix it, although that's a server side problem, so it would go in Bloombox/platform.

Reply:
I am wondering if accepted future/queued orders would make more sense as a status of its own in this case. Perhaps make it inherently accept future orders while the store is an open state. Otherwise, maybe consider a status where the store isn't open, but is taking queued orders for their next open date. (If the store intends to be closed for a while and not accept future orders, they would just be in the CLOSED state)

Reply:
Good point, and by default at the moment, orders are `PENDING` > `ACCEPTED`/`REJECTED` > (some delivery/pickup specific ones) > `FULFILLED`.